hypothesis-python/docs/tutorial/index.rst

```diff
@@ -0,0 +1,17 @@
+Tutorials
+=========
+
+An introduction to Hypothesis for new developers. You may instead want to start with the lightning :doc:`quickstart </quickstart>`, which covers a subset of this material in less detail.
```

Comment:
I think we want this to be more definite: "The Hypothesis tutorial introduces _____.  After completing it, you'll know ___ and be comfortable [doing] ____.  The [quickstart guide] is a much faster bare-bones version if you're in a hurry."

I'd also like to explicitly suggest how readers should engage with this material.  Read through strictly in order (until ___)?  Dip in and out as seems interesting (after ____)?